subsys/testsuite/ztest/src/ztest_new.c

```diff
@@ -181,11 +176,10 @@ void z_impl_z_test_1cpu_start(void)
 	 * to flag the following loop as DEADCODE so suppress the warning.
 	 */
 	/* coverity[DEADCODE] */
-	for (int i = 0; i < NUM_CPUHOLD; i++)  {
-		k_thread_create(&cpuhold_threads[i],
-				cpuhold_stacks[i], CPUHOLD_STACK_SZ,
-				(k_thread_entry_t) cpu_hold, NULL, NULL, NULL,
-				K_HIGHEST_THREAD_PRIO, 0, K_NO_WAIT);
+	for (int i = 0; i < NUM_CPUHOLD; i++) {
+		k_thread_create(&cpuhold_threads[i], cpuhold_stacks[i], CPUHOLD_STACK_SZ,
+				(k_thread_entry_t)cpu_hold, NULL, NULL, NULL, K_HIGHEST_THREAD_PRIO,
+				0, K_NO_WAIT);
```

Comment:
This reminds me that #43609 hasn't merged yet, which is going to collide with the whitespace changes here and require a rebase.  Not a big deal I guess.